README.md

```diff
@@ -1188,40 +1188,40 @@ Use this parameter — especially when using the `fuzzy_match` parameter — to
-  This parameter is required and will accept a string or an array of strings as input.
+  The name of a job. By default, the job names must be exact.
+  See the `fuzzy_match` parameter below for an example of pattern matching job names.
+  This parameter is required and accepts a string or an array of strings as input.
 
 * **fuzzy_match**
 
-  Modifies the behavior of the `job_name` parameter so that the value given is a job name pattern to match.
-  Searches are done using the PowerShell `-match` operator.
-  For example, if the string `backup` is passed to the `job_name` parameter, while `fuzzy_match` is set to true, jobs such as `Daily Backup` and `System Backup` will be matched and started.
+  Modifies the behavior of the `job_name` parameter so that the value given is matched to the job name pattern.
+  The `fuzzy_match` paremeter searches using the PowerShell `-match` operator.
+  For example, when `fuzzy_match` is set to true, if you pass the string `backup` to the `job_name` parameter, jobs such as `Daily Backup` and `System Backup` will be matched and executed.
   This is an optional parameter which accepts either `true` or `false`. Its default value is false.
 
 * **step**
 
-  The step number you would like to execute. Leave blank to execute at the first step.
-  Keep in mind that these job step numbers refer to a zero based array of job step.
-  If you want to execute a job starting at step 5 you need to pass 4 to this parameter.
-  This is because contrary to the SQLServer user interface in many places, the programming objects
+  The step number you want to execute. Leave blank to execute the first step.
+  Note that these job step numbers refer to a zero based array of a job step.
+  To execute a job starting at step 5, pass step 4 to this parameter.
+  This is because contrary to the SQLServer user interface, the programming objects
   this task leverages are zero based.
 
 * **wait**
 
-  This parameter tells the task to wait until a set of jobs has completed and then return job data.
-  All matched jobs are started concurrently and the task waits in a loop for all of them to complete.
-  If this parameter is not set to true, then matched jobs are started and the task returns job data indicating that the job is executing.
-  Keep in mind that jobs that complete rapidly may complete before the task has finished executing.
-  This could result in returned job information indicating the job is currently idle. Please inspect the `lastRunDate` property of the job and the steps to determine if the job was executed successfully.
+  This parameter tells the task to wait until a set of jobs has been completed before returning the job data.
+  All matched jobs start concurrently. The task waits in a loop for all of the jobs to complete.
+  If this parameter is not set to true, matched jobs are started and the task returns job data — indicating that the job has executed.
```

Comment:
I'm worried about the past tense of executed here. If the parameter is not set to true, then when the task returns data the step is likely still executing. Perhaps instead of '...has executed.' we could use '...has started.'?